src/backend/vulkan/src/device.rs

```diff
@@ -597,11 +599,25 @@ impl d::Device<B> for super::Device {
                 .collect::<Box<[_]>>();
 
             inplace_or_alloc_from_iter(dependencies_iter, |dependencies| {
-                let info = vk::RenderPassCreateInfo::builder()
-                    .flags(vk::RenderPassCreateFlags::empty())
-                    .attachments(&attachments)
-                    .subpasses(&subpasses)
-                    .dependencies(&dependencies);
+                const VIEW_COUNT: u32 = 2; // FIXME get from xr
```

Comment:
I'm not entirely sure about how and at what level this should be handled. 

From OpenXR spec: `A typical head-mounted VR system has a view configuration with two views, while a typical phone-based AR system has a view configuration with a single view. A simple multi-wall projection-based (CAVE-like) VR system may have a view configuration with at least one view for each display surface (wall, floor, ceiling) in the room.` ([link](https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#view_configurations)). Also Varjo headset seems to have [4 views](https://developer.varjo.com/docs/native/rendering-to-varjo-headsets) in total, two screens per eye.

Probably the screens might have different resolutions, so the selection of multiview count and different textures should be left for the application?

I tried to pull the configuration info through the `wgpu` <--> `gfx` barrier, but did not manage yet ([link to code section](https://github.com/gfx-rs/wgpu/pull/1387/files#diff-7ce647303d1a7f94da19a5dce513d0029e48163356aac91643e1cd335774d439R32)) but did not manage it yet.

Reply:
gfx-hal can't abstract away multi-viewport rendering entirely, it's too low level for this. The most it can offer is exposing the viewport geometry and providing the way to control the multi-viewport swapchain. The user of gfx-hal, which is wgpu in our case, would then be responsible for using this API and actually do the rendering.